src/main.rs

```diff
@@ -36,6 +36,19 @@ fn main() {
         settings.initialization_parameter = InitParameter::InitializeInternal;
     }
 
+    // don't translate newlines (\r\n <=> \n)
```

Comment:
Does CPython do this? My concern is that as this will have process wide effects, this may have unintended consequences for programs that embed RustPython.

Reply:
> this may have unintended consequences for programs that embed RustPython.

That's why I moved it from `vm::initialize` to main, it should only happen for the actual rustpython binary.

> Does CPython do this?

yup: https://github.com/python/cpython/blob/9b648a95ccb4c3b14f1e87158f5c9f5dbb2f62c0/Python/initconfig.c#L1812-L1817